pkg/doc/verifiable/common.go

```diff
@@ -63,7 +64,7 @@ type typedID struct {
 type RefreshService typedID
 
 // TermsOfUse represents terms of use of Verifiable Credential by Issuer or Verifiable Presentation by Holder.
-type TermsOfUse typedID
+type TermsOfUse interface{}
```

Comment:
I'm not too excited when we need to export types/struct that have interface{}

Can you give more context on this change?

Reply:
It was incorrectly defined before. It can be one or more `TermsOfUse`. Its structure is not precisely defined. [They say](https://w3c.github.io/vc-data-model/#terms-of-use) that **type** field is mandatory and **id** field is optional, and 
> The precise contents of each term of use is determined by the specific termsOfUse type definition.
If we use `typedID` alias for `termsOfUse` and after decoding of VC want to marshal it back, we will loose all the original fields except `type` and `id`.
That was the reason I changed the `TermsOfUse` alias.
We can also stay with `typedID` alias and extend it to keep all extra fields in e.g. `map[string]interface{}` to be used during marshalling.
What do you think @troyronda

Reply:
I thought we had typically used an unexported "raw" struct and filled a properly typed exported struct.

Reply:
I see. It's another situation with extended fields. I would think you would have defined the known parts and keep extra fields in map[string]interface{} like you did elsewhere in the PR.

Reply:
Yes, because of "one or more" and also because of the flexible structure of the `termsOfUse` field.

Reply:
Filed a separate issue #586.